### DISCUSSION TITLE
Converting ETF Listings from USD to EUR in Portfolio Tracking

### DISCUSSION THREAD

**Discussion by Heidieker** [ghostfolio/ghostfolio#3099] [Q&A] [answered]
I have a question for understanding. I have an ETF IE00B4L5Y983, which is listed in USD on Yahoo. I buy it through my trader in EUR, and there it is listed in EUR. Until now, I have entered it in Ghostfolio in USD, because I cannot enter it in EUR. Is there a way to enter it in EUR? The principle of the platforms does not seem to help here.
Do i have a misunderstanding?

This is the same ETF:
USD:
https://finance.yahoo.com/quote/IWDA.L?.tsrc=fin-srch

EUR:
https://www.justetf.com/de/etf-profile.html?isin=IE00B4L5Y983#uebersicht

**Comment by dtslvr** [comment, marked as answer]
Hi @Heidieker

You could convert the exchange rate once when recording the activity. My personal preference is to find the asset in the correct currency ([EUNL.DE](https://finance.yahoo.com/quote/EUNL.DE) in your case?).

**Reply by Heidieker** [reply]
Thank you very much. I tried to find the asset in EUR, but I was unsuccessful. Now it works just the way I wanted it to.
